src/UI/CarouselWrapper.tsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
 import { ComponentBasicProps } from './types';
 
-const CarouselWrapper = ({ children }: ComponentBasicProps): JSX.Element => (
-  <div
-    style={{
-      position: 'relative',
-      display: 'flex',
-      flexDirection: 'column',
-      alignItems: 'center',
-      gap: '20px',
-    }}
-  >
+const CarouselWrapper = ({
+  children,
+  className,
+  ...props
+}: ComponentBasicProps): JSX.Element => (
+  <div className={`slider ${className ?? ''}`} {...props}>
```

Comment:
We need a syntax like className={`slider ${className ?? ''}`} to preserve the class `slider`. If we just write className='Slider' and then use Slider in a styled-component then the classes will be overwritten by styled-components and the styles applied to `slider` will be gone.

Reply:
<div className={`slider ${className}`} {...props}>